src/compose.js

```diff
@@ -7,12 +7,13 @@ const assign = Object.assign;
 
 /**
  * Creates new factory instance.
- * @param {object} descriptor The information about the object the factory will be creating.
+ * @param {Descriptor} descriptor The information about the object the factory will be creating.
  * @returns {Function} The new factory function.
  */
 function createFactory(descriptor) {
   return function Stamp(options, ...args) {
-    const obj = Object.create(descriptor.methods || {});
+    // Next line was optimized for most JS VMs. Please, be careful here!
+    const obj = Object.create(descriptor.methods || null);
```

Comment:
Using `new` would mean something like this if I am not mistaken, but I would be really surprised if it would be really faster.
```js
function createFactory(descriptor) {
  function Stamp(options, ...args) {
    merge(this, descriptor.deepProperties);
    assign(this, descriptor.properties);
    Object.defineProperties(this, descriptor.propertyDescriptors || {});
    ...
  }
  if (descriptor.methods) {
    Stamp.prototype = descriptor.methods;
  }
  return (...args) => new Stamp(...args)
}
```